### ISSUE TITLE
data set repo

### ISSUE THREAD

**Issue by hnegi1212** [abhishekkrthakur/approachingalmost#12]
Hi,

I do understand that code used in book cant be shared, but could you tell us where the data set used in the book can be found.
Its good to have all data sets used in book at one repo, else we have to google each and everytime the dataset being used in book.

**Comment by abhishekkrthakur** [comment]
@hnegi1212 I have provided the references to all datasets when they are discussed the first time. One dataset is not publicly available but that is it. Which dataset were you unable to see the reference too? 
I might also create a markdown with links to datasets used if that makes things easier but I would really like to know which one I missed the reference to?

**Comment by epogrebnyak** [comment]
+1 for _Datasets mentioned_ section in the repo

**Comment by hnegi1212** [comment]
+1 for markdown with links to datasets
Thanks that answers

**Comment by anandsm7** [comment]
+1 for dataset mention
Reference to data set in the Feature Engineering chapter is missing.

**Comment by akagrawal2k17** [comment]
siim-acr-pneumotorax-segmentation dataset in png format (stage 1) is not avaialble. Where can we find that?

**Comment by ranglescabau** [comment]
Yes! I am also struggling to find the correct  siim-acr-pneumotorax-segmentation dataset. :)

**Comment by tokuma09** [comment]
@akagrawal2k17
I found siim-acr-pneumotorax-segmentation dataset.

- png format: [here](https://www.kaggle.com/abhishek/siim-png-images).
- Image id: `train-rle.csv` in [this dataset](https://www.kaggle.com/abhishek/siim-dicom-images)

I don't know whether this is correct, but with a little preprocessing,  my model works well using these two dataset. 

I hope this helps.

**Comment by abhishekkrthakur** [comment]
Is this issue solved now?

**Comment by akagrawal2k17** [comment]
No. I am still looking for the data set and could not find it. Please help me with the dataset.

**Comment by danhphan** [comment]
@akagrawal2k17: For the siim-pneumotorax image classification task, we can download images and train.csv file as followed:

# Download train and test images at https://www.kaggle.com/abhishek/siim-png-images
```
kaggle datasets download -d abhishek/siim-png-images
unzip unzip siim-png-images.zip
```
# Download train.csv with target label at https://www.kaggle.com/abhishek/siim-png-train-csv
```
kaggle datasets download abhishek/siim-png-train-csv
unzip siim-png-train-csv.zip
```

**Comment by CarlosBrunner** [comment]
[Here](https://www.kaggle.com/datasets/abhishek/aaamlp?resource=download&select=adult.csv) are plenty of datasets. Haven't checked if all the datasets used in the book are here though.
